provider/cmd/app/app.go

```diff
@@ -109,6 +111,8 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 
 	flag.StringVar(&virshExecutable, "virsh-executable", "virsh", "Path / name of the virsh executable.")
 
+	fs.BoolVar(&o.DisableHugepages, "disable-hugepages", false, "Disable using Hugepages.")
```

Comment:
Can we please disable the usage of hugepages by default?

Reply:
it is possible. it is copied approach from virtlet.

Reply:
fixed